docs/content/docs/integrations/aks.md

```diff
@@ -4,4 +4,147 @@ description: How to connect to an AKS cluster inside a Porter bundle.
+
+```bash
+export AZURE_CLIENT_ID="insert_ApplicationID"
+export AZURE_TENANT_ID="insert_TenantID"
+export AZURE_CLIENT_SECRET="insert_SecretValue"
+export AZURE_SUBSCRIPTION_ID="insert_SubscriptionID"
+```
+
+### Bundle setup
+
+* After the variables have been set, create a credentials set, by saving the following json to a file e.g. `aks_creds.json` and apply the file: `porter credentials apply aks_creds.json`
+
+```json
+{
+    "schemaType": "CredentialSet",
+    "schemaVersion": "1.0.1",
+    "name": "akscreds",
+    "namespace": "",
+    "credentials": [
+
+        {
+            "name": "azure_client_id",
+            "source": {
+                "env": "AZURE_CLIENT_ID"
+            }
+        },
+        
+        {
+            "name": "azure_client_secret",
+            "source": {
+                "env": "AZURE_CLIENT_SECRET"
+            }
+        },
+
+        {
+            "name": "azure_subscription_id",
+            "source": {
+                "env": "AZURE_SUBSCRIPTION_ID"
+            }
+        },
+
+        {
+            "name": "azure_tenant_id",
+            "source": {
+                "env": "AZURE_TENANT_ID"
+            }
+        }
+    ]
+}
+```
+
+* Verify that `akscreds` credentials set exists: `porter credentials list`
+
+* Install [az mixin](https://porter.sh/mixins/az/) and declare the credentials in the bundle manifest:
+
+```yaml
+credentials:
+- name: azure_client_id
+  env: AZURE_CLIENT_ID
+  description: AAD Client ID for Azure account authentication - used for AKS Cluster SPN details and for authentication to azure to get KubeConfig
+- name: azure_tenant_id
+  env: AZURE_TENANT_ID
+  description: Azure AAD Tenant Id for Azure account authentication - used to authenticate to Azure to get KubeConfig 
+- name: azure_client_secret
+  env: AZURE_CLIENT_SECRET
+  description: AAD Client Secret for Azure account authentication - used for AKS Cluster SPN details and for authentication to azure to get KubeConfig
+- name: azure_subscription_id
+  env: AZURE_SUBSCRIPTION_ID
+  description: Azure Subscription Id used to set the subscription where the account has access to multiple subscriptions
+```
+
+* Declare Azure resource group and AKS name as parameters:
+```yaml
+parameters:
+  - name: rg_name
+    type: string
+    description: "Resource group in which AKS resides"
+    default: "demorg"
+    applyTo:
+      - merge
+      
+  - name: aks_name
+    type: string
+    description: "AKS name"
+    default: "demoaks"
+    applyTo:
+      - merge
+```
+
+* Bellow you can find a custom action called `merge` that it's used to autheticate in Azure and sets the desired subscription (that contains the resource group and AKS cluster) and uses [outputs](https://porter.sh/wiring/#outputs) to save the *kubeconfig* file:
+
+```bash
+merge:
```

Comment:
I think it would make more sense to focus on connecting to AKS inside the bundle, for example by removing the custom action and putting it in the install action instead.